--- a/plugin/data_masking/src/udf/udf_utils.cc
+++ b/plugin/data_masking/src/udf/udf_utils.cc
@@ -186,7 +186,7 @@
 bool Charset_service::set_args_charset(UDF_ARGS *args,
                                        const std::string &charset_name) {
   char *charset = const_cast<char *>(charset_name.c_str());
-  for (uint index = 0; index < args->arg_count; ++index) {
+  for (size_t index = 0; index < args->arg_count; ++index) {
     if (udf_metadata_service->argument_set(args, Charset_service::arg_type,
                                            index,
                                            static_cast<void *>(charset))) {
