--- a/mysys/posix_timers.c
+++ b/mysys/posix_timers.c
@@ -188,10 +188,20 @@
           On error, -1 is returned, and errno is set to indicate the error.
 */

+struct siginfo_sigevent {
+    union sigval sigev_value;
+    int sigev_signo;
+    int sigev_notify;
+    union {
+       int _pad[64-sizeof(int) * 2 + sizeof(union sigval)];
+       int _tid;
+    } _sigev_un;
+};
+
 int
 my_timer_create(my_timer_t *timer)
 {
-  struct sigevent sigev;
+  struct siginfo_sigevent sigev;

   memset(&sigev, 0, sizeof(sigev));

@@ -311,7 +321,7 @@
 int
 my_timer_create(my_timer_t *timer)
 {
-  struct sigevent sigev;
+  struct siginfo_sigevent sigev;
   port_notify_t port_notify;

   port_notify.portnfy_port= port_id;
